script/cms/genBundleSchema.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs-extra');
+const Conf = require('./config');
+const _ = require('lodash');
+
+const conf = new Conf();
+/**
+ * Parse the openAPI document and generate from it a list of bundles
+ * and for each bundles the list of fields.
+ * That should help us when generating new transformers
+ */
+
+const ignoredBundleTypes = [
+  'consumer',
+  'crop',
+  'entity_subqueue',
+  'file',
+  'menu_link_content',
+  'oauth2_token',
+  'path_alias',
+  'redirect',
+  'section_association',
+  'site_alert',
+];
+
+class BundleSchema {
+  constructor() {
+    this.bundles = {};
+    const schema = fs.readJsonSync(conf.cmsExportSchema);
+    this.schema = schema.definitions;
+    this.loadBundles();
+    const originalName = `${conf.cacheDir}/drupal/originalOpenApiSchema.json`;
+    fs.writeFileSync(originalName, JSON.stringify(schema, null, 2));
+    /* eslint-disable no-console */
+    console.log(`Saved original openAPI schema to  ${originalName}`);
+    const prettyOutJson = JSON.stringify(this.bundles, null, 2);
+    const fileName = `${conf.cacheDir}/drupal/openApiBundleSchema.json`;
+    fs.writeFileSync(fileName, prettyOutJson);
+    /* eslint-disable no-console */
+    console.log(`Saved to ${fileName}`);
+  }
+
+  /**
+   * load the openAPI exported json file and save the part we're interested in, the bundles
+   * info into a file fullSchema.json
+   */
+  saveSchema() {
+    const prettyOutJson = JSON.stringify(this.schema, null, 2);
+    fs.writeFileSync('fullSchema.json', prettyOutJson);
+  }
+
+  /**
+   * Find the names of the bundles in the schema
+   * info into a file schema.json
+   */
+  loadBundles() {
+    let ii = 0;
+    _.map(this.schema, (value, key) => {
```

Comment:
Can this be accomplished with `this.schema.forEach()`? (Or perhaps `Object.entries(this.schema).forEach()` if `this.schema` is an object.)